### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Dr-Ozzey** [spimort/TerraBrush#75] [Q&A]
Hi guys,
After installing plugin and trying tick it in project settings I get an error: "No loader found for resource: res://addons/terrabrush/Plugin.cs".
Same error for Godot 4.1 & 4.2. How to fix it? Am I doing something wrong?

**Comment by niklasking** [comment]
Have you compiled the project before activating the plug-in?

**Reply by Dr-Ozzey** [reply]
Compiled? Or you mean export the project? I have never used any Godot plugins, so I may di something wrong.

**Reply by Dr-Ozzey** [reply]
Ok, I know what do you mean. I have read the installation manual. Now, I know what to do. Thanks

**Reply by spimort** [reply]
Awesome! I'll close the discussion then 🙂
